docs/articles/documentation/test-api/actions/take-screenshot.md

```diff
@@ -6,23 +6,99 @@ checked: true
 ---
 # Take Screenshot
 
-Takes a screenshot of the tested page.
+This topic describes how you can take screenshots of the tested page.
 
-**Note**: this action requires a [ICCCM/EWMH-compliant window manager](https://en.wikipedia.org/wiki/Comparison_of_X_window_managers) on Linux.
+**Note**: these actions require a [ICCCM/EWMH-compliant window manager](https://en.wikipedia.org/wiki/Comparison_of_X_window_managers) on Linux.
+
+> Important! If the screenshot directory is not specified with the [runner.screenshots](../../using-testcafe/programming-interface/runner.md#screenshots) API method or the [screenshots](../../using-testcafe/command-line-interface.md#-s-path---screenshots-path) command line option,
```

Comment:
Do we have any warning messages about that peculiarity during test run?

Reply:
Yes https://github.com/DevExpress/testcafe/blob/master/src/notifications/warning-message.js#L2

Reply:
Yes, we have